README.md

```diff
@@ -140,6 +140,7 @@ to build others on top. Feel free to PR more that are missing.
 * `cenc.raw.json` - Encodes a JSON value as utf-8 without a length prefixed.
 * `cenc.ndjson` - Encodes a JSON value as newline delimited utf-8.
 * `cenc.raw.ndjson` - Encodes a JSON value as newline delimited utf-8 without a length prefixed.
+* `cenc.any` - Encodes any value into a buffer - self described. Like JSON+buffer, but using compact types.
```

Comment:
```suggestion
* `cenc.any` - Encodes any JSON representable value into a self described buffer. Like JSON + buffer, but using compact types. Useful for schemaless codecs.
```

Trying to qualify that it's not literally any value, but some reasonable subset of JavaScript values. A comparison to BSON or MessagePack might also be helpful?